index.html

```diff
@@ -587,6 +795,16 @@ <h3>
             </ol>
           </li>
         </ol>
+        <p>
+          To <dfn>normalize a logical axis value</dfn> given |value|, |minimum|, and |maximum|:
+        </p>
+        <ol>
+          <li>
+            Return 2 (|value| − |minimum|) / (|maximum| − |minimum|) − 1.
```

Comment:
Should not it be this, so that the normalized output of input with min `1`, max `5` and val `3` returns `1/2`?
```suggestion
            Return (|value| − |minimum|) / (|maximum| − |minimum|).
```